docs/source/customizing.ipynb

```diff
@@ -279,7 +316,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.4.3+"
+   "version": "3.4.3"
```

Comment:
The `+` shouldn't really have been here. Possibly we should be more careful about how we get the Python version number from `sys`.